### ISSUE TITLE
Make changes to Buffer ussage

### ISSUE THREAD

**Issue by tshemsedinov** [metarhia/Impress#513]
```JavaScript
new Buffer(size); // Old
Buffer.alloc(size); // New

new Buffer(str, 'utf8'); // Old
Buffer.from(str, 'utf8'); // New
```

**Comment by tshemsedinov** [comment]
We should do that in separate branch @aqrln

**Comment by aqrln** [comment]
@tshemsedinov whether it should be done in `v0.2` branch or a completely separate branch for Node 6.x?

**Comment by tshemsedinov** [comment]
Completely separate, forked from master, and to be merged into 0.2
